scripts/stac/imagery/capture_aera.py

```diff
@@ -0,0 +1,68 @@
+import json
+from typing import Any, Dict, List
+
+from shapely import BufferCapStyle, BufferJoinStyle, Geometry, to_geojson, union_all
+from shapely.geometry import Polygon
+
+DECIMAL_DEGREES_1M = 0.00001
+"""
+Degree precision of ~1m (decimal places 5, https://en.wikipedia.org/wiki/Decimal_degrees)
+"""
+
+
+def to_feature(geometry: Geometry) -> Dict[str, Any]:
+    """Transform a Geometry to a GeoJSON feature.
+
+    Args:
+        geometry: a Geometry
+
+    Returns:
+        a GeoJSON document.
+    """
+    return {"geometry": json.loads(to_geojson(geometry)), "type": "Feature", "properties": {}}
+
+
+def merge_polygons(polygons: List[Polygon], buffer_distance: float) -> Geometry:
+    """Merge a list of polygons by converting them to a single geometry that covers the same area.
+    A buffer distance is used to buffer out the polygons before dissolving them together and then negative buffer them back in.
+    The merged geometry is simplify (rounded) to the decimal used for the buffer.
+
+    Args:
+        polygons: list of polygons to merge
+        buffer_distance: decimal places to use to buffer the polygons
+
+    Returns:
+        A single Geometry.
+    """
+    buffered_polygons = []
+    for poly in polygons:
+        # Buffer each polygon to round up to the `buffer_distance`
+        buffered_poly = poly.buffer(buffer_distance, cap_style=BufferCapStyle.flat, join_style=BufferJoinStyle.mitre)
+        buffered_polygons.append(buffered_poly)
+    union = union_all(buffered_polygons)
+    # Negative buffer back in the polygons
+    union = union.buffer(-buffer_distance, cap_style=BufferCapStyle.flat, join_style=BufferJoinStyle.mitre)
+    union = union.simplify(buffer_distance)
```

Comment:
Could we come up with some different names for the three `union`s to clarify the intermediate results? Maybe something like `raw_union`, `unbuffered_union`, and `simplified_union`?